### PR TITLE
Upgrade social-auth-app-django -- BOM-1216

### PR DESCRIPTION
**Pull request by iamsobanjaved**
JRA: [BOM-1216](https://openedx.atlassian.net/browse/BOM-1216)

- Upgrade socail-auth-app-django

### CODE REVIEW

**Review by Ayub-Khan — APPROVED** [review]
LGTM

File: requirements/e2e.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ pyparsing==2.4.6          # via packaging
 pytest-base-url==1.4.1    # via pytest-selenium
 pytest-html==1.22.1       # via pytest-selenium
 pytest-metadata==1.8.0    # via pytest-html
-pytest-randomly==3.2.1
+pytest-randomly==1.2.3
```
Comment (jmbowman) [inline requirements/e2e.txt]: Weird, any idea what caused this to be downgraded?

**Review by jmbowman — APPROVED** [review]

File: requirements/e2e.txt (hunk 1 above)
Comment (iamsobanjaved) [reply requirements/e2e.txt]: This line `pytest-randomly < 2.0.0  # Pin below 2.0 until we're on python 3` in _e2e.in_ caused this. We don't need this constraint now ?

Comment (jmbowman) [reply requirements/e2e.txt]: We shouldn't, and I'm not sure how we ended up on the newer version with that there.  Anyway, thanks for removing the constraint.

**Pull request merged**